lib/appnexusapi/service.rb

```diff
@@ -54,6 +55,19 @@ def get(params={})
     end
   end
 
+  def get_all(params = {})
+    responses = []
+    last_responses = get(params)
+
+    while last_responses.size > 0
+      responses += last_responses
+      last_responses = get(params.merge('start_element' => responses.size))
+      sleep(1) # The gem has no error handling at all for rate limit errors; sleeping for a second prevents errors
```

Comment:
I'm pretty sure the rate limit is more than 60 per minute, so it would be nice if this was configurable or calculated based on the rate limit. thoughts?